packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/resolve_import_errors.ts

```diff
@@ -235,6 +243,7 @@ export async function resolveSavedObjectsImportErrors({
           ...(overwrite && { overwrite }),
           ...(destinationId && { destinationId }),
           ...(destinationId && !originId && !createNewCopies && { createNewCopy: true }),
+          ...{ managed: createdObject.managed ?? managed ?? false }, // double sure that this already exists but doing a check just in case
```

Comment:
NIT: Yeah, I think `managed: createdObject.managed` would be sufficient (and we don't need the spread operator here given the value will always be present)

Reply:
As per the comment: I was worried about missing some indirect method that's called. We can change that later if needs be,